include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -1421,7 +1421,9 @@ class CommonOperatorLoader : public ProtobufLoader {
   }
 
   // Loads Less operator. Internally it's a cmpLT Node.
-  Error loadLess(const OpType &op, ArgumentDictionaryTy &dict) {
+  // Reused for GreaterOrEqual by setting swapInputs to true.
+  Error loadLess(const OpType &op, ArgumentDictionaryTy &dict,
+                 bool swapInputs = false) {
```

Comment:
I'm wondering maybe instead we can create a new function in ONNXModelLoader.cpp that will cover loading GreaterOrEqual operator to minimize changes in this file.

Reply:
@katayoun-cadence Thanks for your review. I preferred it this way, because then this can also be extended to support GE operator for caffe2 in future ( which was not my immediate need). In fact it can be extended to any "common" complimentary operator where we can just use swap flag to implement 2 operators through just one common function. That's my preference.

Reply:
Maybe we could have some generic function instead like below?
```
template <typename CmpType>
Error loadCmpOp(const OpType &op, ArgumentDictionaryTy &dict, bool swapInputs = false) {
    // Input Type.
    NodeValue xNV;
    ASSIGN_VALUE_OR_RETURN_ERR(xNV, getNodeValueByName(op.input(0)));
    NodeValue yNV;
    ASSIGN_VALUE_OR_RETURN_ERR(yNV, getNodeValueByName(op.input(1)));
    std::string opName = loadOperatorName(op);
    auto *xNode = xNV.getNode();
    auto *yNode = yNV.getNode();

    Node *N = swapInputs ? G_->createNodeWithBroadcast<CmpType>(
                               opName, /* axis */ -1, yNode, xNode)
                         : G_->createNodeWithBroadcast<CmpType>(
                               opName, /* axis */ -1, xNode, yNode);

    RETURN_IF_ERR(addNodeAsOutput(op, N));
    return Error::success();
}
```

So, then we can avoid having that boolean flag `swapInputs` in `loadLess` to make sure we don't necessarily extend its semantics. E.g. we could have then:
```
Error loadLess(const OpType &op, ArgumentDictionaryTy &dict) {
  return loadCmpOp<CmpLTNode>(op, dict, /* swapInputs */ false);
}

Error loadGreaterOrEqual(const OpType &op, ArgumentDictionaryTy &dict) {
  return loadCmpOp<CmpLTNode>(op, dict, /* swapInputs */ true);
}
```

How does that sound?

Reply:
sounds good.